### PR TITLE
feat: Prepend network if symbol already exists

### PR DESCRIPTION
**Pull request by AndonMitev**
# :books: https://linear.app/liquality/issue/LIQ-578/add-message-for-editing-token-symbol-luna-while-adding-the-token-for :books:

### CODE REVIEW

**Review by beemi — APPROVED** [review]

File: src/views/CustomToken.vue [hunk 1]
```diff
@@ -136,11 +136,18 @@ export default {
     networkAssets () {
       return this.enabledAssets[this.activeNetwork][this.activeWalletId]
     },
-    isExistingNetworkAsset () {
-      return Boolean(this.networkAssets.find(_symbol => _symbol === this.symbol))
+    chainNativeAsset() {
+      return {
+        ethereum: 'ETH',
```
Comment (skeremidchiev) [inline src/views/CustomToken.vue:141]: It's a better approach to get chains from cryptoassets.

Comment (AndonMitev) [reply src/views/CustomToken.vue]: fixed

**Review by skeremidchiev — APPROVED** [review]

File: src/views/CustomToken.vue [hunk 2]
```diff
@@ -214,8 +218,9 @@ export default {
         this.isSymbolEditable = Boolean(this.networkAssets.find(_symbol => _symbol === this.symbol))
       }
     }, 500),
-    async selectChain (chain) {
+    async selectChain (chain, nativeAsset) {
       this.chain = chain
+      this.nativeAsset = nativeAsset
```
Comment (bradleySuira) [inline src/views/CustomToken.vue:223]: Hi, I think that we don't need to set the nativeAsset manually, we can get it from cryptoassets:
```
import { chains } from '@liquality/cryptoassets'
...
async selectChain (chain) {
 this.chain = chain
 this.nativeAsset = nativeAsset
 const { nativeAsset } = chains[nativeAsset]
 this.nativeAsset = nativeAsset
...
}
```

**Review by bradleySuira — CHANGES_REQUESTED** [review]

**Comment by beemi** [comment]
@AndonMitev  can u please last comment

**Pull request closed without merging**